profiles/base/package.use.mask

```diff
@@ -3,6 +3,10 @@
 
 # New entries go on top.
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2025-01-22)
+# dev-qt/qtbluetooth:5 removal for CVE-2025-23050, bug #948573.
+dev-python/qtpy5 bluetooth
```

Comment:
Typo obv, but alternatively can add https://termbin.com/jx9p to this PR to be spared cleaning up profile later